engine/executor/dm/worker.go

```diff
@@ -103,6 +113,7 @@ func newDMWorker(ctx *dcontext.Context, masterID libModel.MasterID, workerType l
 
 // InitImpl implements lib.WorkerImpl.InitImpl
 func (w *dmWorker) InitImpl(ctx context.Context) error {
```

Comment:
If InitImpl meets error, InitImpl itself should be responsible to clean the started components?

Reply:
cc @liuzix @amyangfei